### ISSUE TITLE
The plugin "gatsby-source-plone" deleted a node of a type owned by another plugin

### ISSUE THREAD

**Issue by datakurre** [collective/gatsby-source-plone#202]
I've now seen twice this error when deleting a deep folder structure from the example project while running gatsby develop:

```
  Error: The plugin "gatsby-source-plone" deleted a node of a type owned by another plugin.
            The node type "PloneFolder" is owned by "undefined".
            The node object passed to "deleteNode":
```

This does not make sense from this source plugin point of view, but is probably an issue related to GatsbyJS cache.

I'll leave this issue open here for further observations...

**Comment by iFlameing** [comment]
@datakurre can you give me folder structure so that I can also reproduce it?

**Comment by datakurre** [comment]
@iFlameing It's the default example content.

Run ``make init-backend``.

Run ``make watch`` and leave it running.

Delete /Plone/examples -folder.

Run ``make init-backend`` again to allow content to reappear.

Delete /Plone/examples -folder again.

If you still don't get the error, try to break "make watch" and run it again so that gatsby develop starts up with the cache.

**Comment by iFlameing** [comment]
@datakurre thanks!

**Comment by datakurre** [comment]
I just show this "is owned by "undefined"" error also when trying to delete a single page. Stopping gatsby-develop, running "make init-backend", removing GatsbyJS .cache and public (make clean) fixed the issue.

So definitely something gets wrong in GatsbyJS cache, but I'm not sure by whose fault.

**Comment by datakurre** [comment]
I believe that node owner information is something that should be managed by GatsbyJS.

**Comment by datakurre** [comment]
Found the reason for this issue: https://github.com/gatsbyjs/gatsby/issues/15918

**Comment by datakurre** [comment]
This should be fixed with Published gatsby@2.13.34
